test/unit/evaluator_spec.js

```diff
@@ -1,8 +1,46 @@
-/* globals OperatorList, WorkerTask, PartialEvaluator, StringStream, OPS, Dict,
-           Name, Stream */
-
+/* Copyright 2017 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 'use strict';
 
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    define('pdfjs-test/unit/evaluator_spec', ['exports',
+      'pdfjs/core/evaluator', 'pdfjs/core/primitives', 'pdfjs/core/stream',
+      'pdfjs/core/worker', 'pdfjs/shared/util'], factory);
+  } else if (typeof exports !== 'undefined') {
+    factory(exports, require('../../src/core/evaluator.js'),
+      require('../../src/core/primitives.js'),
+      require('../../src/core/stream.js'), require('../../src/core/worker.js'),
+      require('../../src/shared/util.js'));
+  } else {
+    factory((root.pdfjsTestEvaluatorSpec = {}), root.pdfjsCoreEvaluator,
+      root.pdfjsCorePrimitives, root.pdfjsCoreStream, root.pdfjsCoreWorker,
+      root.pdfjsSharedUtil);
+  }
+}(this, function (exports, coreEvaluator, corePrimitives, coreStream,
+  coreWorker, sharedUtil) {
```

Comment:
Nit: Please change the indentation of this line so that the parameter names line up, compare with e.g. [this file](https://github.com/mozilla/pdf.js/blob/aabfb7788ad140c424dd0b7823f35255d13f5225/src/display/api.js#L35-L36); it should look like this:
```javascript
}(this, function (exports, coreEvaluator, corePrimitives, coreStream,
                  coreWorker, sharedUtil) {
```